### ISSUE TITLE
ember-cli-blanket does not work with ember-cli-qunit 2.2.2 or newer.

### ISSUE THREAD

**Issue by nathanhammond** [sglanzer/ember-cli-blanket#162]
The fix here is for `ember-cli-blanket` to hook [`ember-cli-test-loader`](https://github.com/ember-cli/ember-cli-test-loader) to insert its module instrumentation. That's the ideal insertion point for what you're trying to accomplish. This also has the benefit of reducing the number of intimate and private APIs inside of Ember's module loading infrastructure that it needs to touch. Also it no longer has to support `autostart` as a feature since modules will be instrumented as part of the same loading step that Ember normally uses.

**Comment by rwwagner90** [comment]
👍 definitely broken for all my addons that I have updated 😞

**Comment by sglanzer** [comment]
I'd love to find time to fix this, but most of my effort on code coverage has been dedicated to `ember-cli-code-coverage` these days.  I'm more than happy to accept a PR, but I doubt I'll have time to fix this for at least a week or two.

**Comment by luketheobscure** [comment]
@sglanzer should we start migrating our projects over to ember-cli-code-coverage then?

**Comment by sglanzer** [comment]
@luketheobscure Yeah, with the 0.3 release `ember-cli-code-coverage` supports addons, so I would encourage a swap.  I haven't marked this repo as deprecated yet, but that's probably the next step.  I don't think @jschilli or I have any time to actively work on maintaining this and I believe `ember-cli-code-coverage` is the way forward

**Comment by sglanzer** [comment]
@jschilli @rwjblue @kategengler any thoughts on endorsing a cutover?

**Comment by luketheobscure** [comment]
@sglanzer It would still be helpful to get this fixed. I can't switch over to `ember-cli-code-coverage` until they get it playing nicely with CoffeeScript (https://github.com/kategengler/ember-cli-code-coverage/issues/55).

**Comment by rwwagner90** [comment]
@luketheobscure I think we ought to have everyone focus on ember-cli-code-coverage, rather than putting time into trying to get this working again.

**Comment by sglanzer** [comment]
@luketheobscure I have to agree with @rwwagner90 - unfortunately the time available to dedicate to the type of project is severely limited, so we need to throw any effort behind a blessed implementation

I'm also more than happy to accept a PR and publish a new release if you find that to be a shorter path for resolving your situation

**Comment by RobbieTheWagner** [comment]
Should we close this issue, since ember-cli-code-coverage is the new thing to use?

**Comment by sglanzer-fire** [comment]
Yeah, I'd be in favor of archiving the project as a whole

**Comment by RobbieTheWagner** [comment]
@sglanzer deprecating does sound like a good idea. I was just looking through open issues that are lingering in my GitHub and wanted to see what you thought.
